P5/Source/Guidelines/en/DI-PrintDictionaries.xml

```diff
@@ -2658,25 +2658,28 @@ equivalently "poj." in Polish, or "Ez." in German, etc., what is actually referr
 grammatical value that can be rendered with a plethora of markers, depending on the publisher, language, or lexicographic tradition.
 In order to signal that this variety of surface markers in fact indicate the same
 underlying value, it is possible to align them with an external inventory of standardized
-values. The TEI provides means to align grammatical categories as well as their content
-with the ISOcat reference, which is a Web implementation of <ref target="#ISO-12620">ISO 12620</ref>.</p>
+values. The TEI provides the <ident type="class">att.datcat</ident> attribute class for the purpose of aligning grammatical (or indeed any sort of) categories as well as their values
+with a reference taxonomy of shared data categories.</p>
 <p>In the example below, a fragment of the entry for <foreign>isotope</foreign> cited
-in section <ptr target="#DITPGR"/> is adorned by references to ISOcat definitions for "part
-of speech" (<att>dcr:datcat</att>) and "adjective" (<att>dcr:valueDatcat</att>). Depending
+in section <ptr target="#DITPGR"/> is adorned by references to external definitions for "part
+of speech" (<att>datcat</att>) and "adjective" (<att>valueDatcat</att>). Depending
 on the status and extent of the dictionary, various strategies may be used to reduce the
-redundancy of the repeated ISOcat references.<egXML xmlns="http://www.tei-c.org/ns/Examples" valid="feasible">
-<entry xmlns:dcr="http://www.isocat.org/ns/dcr">
+redundancy of the potentially redundant references.<egXML xmlns="http://www.tei-c.org/ns/Examples" valid="feasible">
```

Comment:
"redundancy" and "redundant" in one sentence. Is "potentially redundant" really needed here? I think the sentence is clear without it.

Reply:
true, fixed